### ISSUE TITLE
Ambiguity in the mechanism for saving bin stats for emnist data

### ISSUE THREAD

**Issue by rahulv0205** [cianeastwood/bufr#5]
In section 3.1 of the paper, it is written that features  are extracted from feature extractor part g_s. 
<img width="729" alt="Screenshot 2023-04-07 at 7 17 02 PM" src="https://user-images.githubusercontent.com/102671411/230620111-64cc22e5-12a0-41fb-9318-424e7056a037.png">

After pretraining, when we run the save_train_stats.py file for EMNIST Dataset, the updated model after adding softbinstats layers look like: 
<img width="783" alt="Screenshot 2023-04-07 at 7 20 21 PM" src="https://user-images.githubusercontent.com/102671411/230620071-1512772b-46b1-4993-b5e4-0be80cfc34ce.png">

We can clearly see that layers are being added into classifier part instead of feature extractor part. How is this layer capturing the features even when it is in the classifier part? Or is there any misinterpretation in my understanding?

**Comment by cianeastwood** [comment]
Hi,

Yes I see there is some ambiguity in the term "classifier".

In the paper, we mean the linear head or *last* linear layer.

In the code, classifier is used to mean *all* of the linear or
fully-connected layers at the end of the network.

Hence, the "features" (paper) are the output of the first linear layer in
"classifier" (code), and the "logits" (paper) are the output of the second
linear layer in "classifier" (code).

Hope this clarifies things!

**Comment by rahulv0205** [comment]
Yeah, thanks for early responding.
